css/class-naming-conventions/readme.md

```diff
@@ -123,21 +123,86 @@ Sometimes you'll write a modifier for a component and you want that modifier to
 </div>
 ```
 
+
+#### 1. Nested inside the component
+Nest the `.c-component__sub-component` elements inside the `.c-component` SCSS.
+
+This method works well when a modifier requires simple changes to a small number of sub-components.
+
 ```scss
 .c-blog-post {
     &.c--featured {
-        [STYLES]
+        ...
+
+        .c-blog-post__title {
+            ...
+        }
     }
 }
+```
+
+
+#### 2. Root level as a selector chain
+Use a selector chain with `.c-component.c--modifier`, and nest the `.c-component__sub-component` elements within it.
+
+This method works especially well when a component has multiple modifiers or state classes that can be coupled with it.
+
+```scss
+.c-blog-post.c--featured {
+    ...
+
+    .c-blog-post__title {
+        ...
+    }
+}
+```
+
+When using the above methods in larger files, add a comment to the `.c-component__sub-component` like so:
+```scss
+// Blog Post Title
+// ---
+//
+// Modifier styles can be found in:
+// .c-blog-post.c--featured
```

Comment:
For small and medium-sized files (ideally most of them) I doubt the cross-reference is needed, so as long as this is a recommendation rather than a requirement, I’d cool with it.